src/common/string/format-list.ts

```diff
@@ -0,0 +1,25 @@
+import memoizeOne from "memoize-one";
+import { HomeAssistant } from "../../types";
+import "../../resources/intl-polyfill";
+
+export const formatListWithAnds = (hass: HomeAssistant, list: string[]) =>
```

Comment:
Use `hass.locale` instead of `hass`
```suggestion
export const formatListWithAnds = (locale: FrontendLocaleData, list: string[]) =>
```